workspaces/orchestrator/plugins/orchestrator-form-widgets/http-workflow-dev-server/exampleWorkflows/schemas/dynamic-course-select__main-schema.json

```diff
@@ -31,6 +31,28 @@
       "title": "This title will never be displayed. This 'courseDetails' property is just a placeholder to be replaced by the 'mySchemaUpdater' based on the fetched response. Will contain complex data later.",
       "ui:widget": "hidden"
     },
+    "suggestedCourses": {
+      "type": "string",
+      "title": "Suggested Additional Courses",
+      "ui:widget": "StaticText",
+      "ui:props": {
+        "ui:variant": "h1",
```

Comment:
We plan to use this templates for demos. Can this be changed to e.g. `h4` and the later to `subtitle1`?

Reply:
dome